docs/source/internals.rst

```diff
@@ -59,17 +63,17 @@ The file names do not need to have the shown names, they can be choosen freely.
 
         // return public methods
         return {
-            load_ipython_extension : load_ipython_extension
+            load_jupyter_extension : load_jupyter_extension
```

Comment:
Unfortunately, I think notebook still only looks for `load_ipython_extension` - so although the function may be named `load_jupyter_extension`, we also need to list it under both keys